frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx

```diff
@@ -91,10 +90,17 @@ function ExplicitSize<T>({
 
       _getElement() {
         try {
-          let element = ReactDOM.findDOMNode(this);
-          if (selector && element instanceof Element) {
+          let element: HTMLElement | null = null;
+          const { forwardedRef } = this.props;
+
+          if (forwardedRef && typeof forwardedRef === "object") {
+            element = forwardedRef.current as HTMLElement;
+          }
```

Comment:
I wonder if the forwarded ref could end up not being a HTML element. This is probably the only way to do it though, aside from adding a container div and attaching a ref to it.